### ISSUE TITLE
Determining work needed from DAC Audit May 2023

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-frontend#3592]
## What

We've requested an external accessibility audit with the Digital Accessibility Centre (DAC) that is due to take place on the 9th May 2023. The audit is scheduled to take 3.5 days in total. 

## Why

This WCAG 2.2 audit supports our efforts to https://github.com/alphagov/govuk-frontend/issues/3040.

As we approach [the publishing date for WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/), we want to ensure our existing govuk-frontend elements are meeting all WCAG 2.1 AA criteria. The only way to pass WCAG 2.2 AA is to also meet all criteria in WCAG 2.1 AA ([with one exception, where a criterion is being removed](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#changes-from-wcag-21-to-wcag-22)).
We're doing other work to assess where updates are needed to align elements with the 9 new WCAG 2.2 criteria, but we likely don't have capacity to run an internal audit across all of the existing 78 WCAG criteria.

## Who needs to work on this
@CharlotteDowns, @davidc-gds

## Who needs to review this
@stevenjmesser

## Done when
- [x] Collect and type up [notes from DAC visit](https://docs.google.com/document/d/1LKyn4OhYLPHIz13oh3FYZB1D6R_HxTy0SEK_P9TfAk4/edit#)
- [x] Capture any updates to audit example based upon notes from visit
- [x] Reach out to DAC to check on progress
- [x] Received [audit result from DAC](https://drive.google.com/file/d/12aMGBqVuZMR3q1T1f05kZmA19ek6oghU/view?usp=drivesdk)
- [x] Create new audit tags (audit may 2023) in both the Design System and Frontend repos
- [x] Add fails as distinct issues on the Frontend repo and Design system repo, applying tag
- [x] Add usability issues as distinct issues on the Frontend repo and Design system repo, applying tag
- [x] Determine whether we can publish the audit paper publicly (chat with Kelly and Steve)
- [x] Add comments to respective component/pattern GitHub issues
- [x] Contact teams who have fails with results of audit
- [ ]  Consider and make any updates to the Design System accessibility statement
- [x]  Add bonus Design System issue to repo

## Links to tagged issues
- [Issues in Frontend repo](https://github.com/alphagov/govuk-frontend/labels/audit%20may%202023)
- [Issues in Design System repo](https://github.com/alphagov/govuk-design-system/labels/audit%20may%202023)

**Comment by CharlotteDowns** [comment]
Some highlights from DAC visit mentioned in [Show and Tell]( https://docs.google.com/presentation/d/1BL8aSIemYOz7xrXk0RgM7GP23i5D021qEd0lA8n-1d4/edit).

**Comment by CharlotteDowns** [comment]
[Issues identified in a spreadsheet format](https://docs.google.com/spreadsheets/d/1qiWCRwH7QofQgJ9iDLRSsfETLKlZCyrv7z04qjZurhM/edit?usp=sharing)

**Comment by CharlotteDowns** [comment]
[Draft of GitHub issue comments for each backlog item](https://docs.google.com/document/d/1BlKiEo_VPm0XUy8fB9rbXvi6Su53E1G8Yalhmp0Q3_Q/edit#)

**Comment by davidc-gds** [comment]
This work is now complete, and is followed by a new issue:
- https://github.com/alphagov/govuk-frontend/issues/3814
